app/services/db/monthly_sp_auth_count/total_monthly_auth_counts_within_iaa_window.rb

```diff
@@ -125,7 +125,7 @@ def partial_month_subqueries(issuer:, partial_months:)
             , COUNT(sp_return_logs.id) AS auth_count
             FROM sp_return_logs
             WHERE
-                  sp_return_logs.requested_at BETWEEN %{range_start} AND %{range_end}
+                  requested_at::date BETWEEN %{range_start} AND %{range_end}
```

Comment:
can we keep the table name in here?
```suggestion
                  sp_return_logs.requested_at::date BETWEEN %{range_start} AND %{range_end}
```